chi_edge/vendor/FATtools/README.MD

```diff
@@ -0,0 +1,164 @@
+FATtools3
+=========
+
+Manually install it under site-packages.
+
+Born to re-sort directory entries in a FAT32 root table to cope with some hardware MP3 players' limits, it now provides full read/write support in Python 3 (both 32- and 64-bit) for FAT12/16/32 and exFAT filesystems, for hacking and recovering purposes.
+
+It is able to open disk partitioned with both MBR or GPT and to help in partitioning.
+It can transparently create, read and write Dynamic and Differencing VHD, VHDX[1], VDI and VMDK disk images.
+It can handle RAW disk images and BytesIO "RamDisk" images, also.
+
+Actually, following features are implemented (mostly in Python, with a few ctypes calls to handle Win32 disks natively; compatibility with Linux is not regularly tested):
```

Comment:
What platforms have we tested this on? This is hopefully a very simple / static use, but I've gotten requests to use the following systems to enroll other devices:
- windows 10
- mac os
- ubuntu, x86_64
- ubuntu aarch64 (jetson)
- raspbian aarch64
- raspbian arm32

Reply:
I tested the library on mc os and ubuntu x86. I imagine the any incompatibility is regarding "real disks" and not images.